docs/dashboard/import-export.md

```diff
@@ -0,0 +1,29 @@
+# Altis Dashboard Import Export
+
+The Altis Dashboard allows you to perform data migrations between environments, as well as perform data exports for use in local development environments.
+
+You can find the Import & Export features under the Data tab of the selected Altis Environment.
+
+![import export view](../assets/import-export-view.png)
+
+From here you can view and download previous exports, displayed as a list, ordered by date. 
+
+You can also create a new export of your database and/or assets. Typically, this will be your WordPress Uploads directory, comprising your media uploads and any other assets your application store there.
+
+To create a new export, you can press the export button. This will display a new pop-up (as shown below) in which you can choose to export the uploads and/or the database. If you choose to create an export of the Uploads, you’ll be able to optionally specify a path. This is useful if you’d quickly like to create an export of assets in a specific directory.
+
+![export modal](../assets/export-modal.png)
+
+You can also import your database and assets from one environment to another, via the Altis Dashboard.
+
+To do this, in the Altis Dashboard, first go to the environment you want to import into. Then select Import. A pop-up form will appear. You can select the source environment you will import from. You can choose database and/or uploaded assets. And, as mentioned above, you can select a sub-path of the assets, if desired.
```

Comment:
To do this, in the Altis Dashboard, first go to the environment you want to import into. Then select Import. A pop-up form will appear. ~You can s~ Select the source environment you will import from. ~You can c~ Choose database and/or uploaded assets. ~And, a~ As mentioned above, you can select a sub-path of the assets, if desired.